doc/api/n-api.md

```diff
@@ -1117,10 +1117,10 @@ NAPI_NO_RETURN void napi_fatal_error(const char* location,
 
 * `[in] location`: Optional location at which the error occurred.
 * `[in] location_len`: The length of the location in bytes, or
-  `NAPI_AUTO_LENGTH` if it is null-terminated.
+  `NAPI_AUTO_LENGTH` if it is `NULL`-terminated.
```

Comment:
This does **not** refer to `NULL` as in a null pointer. Null-terminated is the right word here.